TESTS/mbed_hal_fpga_ci_test_shield/gpio/main.cpp

```diff
@@ -142,31 +162,51 @@ void test_explicit_input(PinName pin)
     TEST_ASSERT_NOT_EQUAL(0, gpio_is_connected(&gpio));
     tester.gpio_write(MbedTester::LogicalPinGPIO0, 0, false);
     wait_us(HI_Z_READ_DELAY_US);
+    // Some targets don't support pull-up/pull-down mode.
+#if !defined(TARGET_NANO100) &&         \
+    !defined(TARGET_NUC472) &&          \
+    !defined(TARGET_M451)
```

Comment:
Since every step of this test (`test_explicit_input()`) sets one of the `PinModes`, the whole test should be skipped if not supported. I think the `#if` statement should be moved to the declaration of `cases` below.

Reply:
@fkjagodzinski Fixed as commented